cron/class-wcal-cron.php

```diff
@@ -609,7 +609,10 @@ public static function wcal_update_status_of_guest( $cart_id, $abandoned_cart_ti
 				$order_date_with_time = $results_query_email[0]->post_date;
 				$order_date           = substr( $order_date_with_time, 0, 10 );
 
-				if ( $order_date == $todays_date ) { // phpcs:ignore
+				if ( 'wc-pending' === $results_query_email[0]->post_status || 'wc-failed' === $results_query_email[0]->post_status && strtotime( $order_date_with_time ) > $abandoned_cart_time ) {
```

Comment:
I think it's best to enclose the condition `'wc-pending' === $results_query_email[0]->post_status || 'wc-failed' === $results_query_email[0]->post_status` inside parenthesis.

Reply:
Fixed